src/client/components/UserPage/Drawer/ControlPanel/index.tsx

```diff
@@ -31,27 +36,64 @@ const useStyles = makeStyles(() =>
     dialogContents: {
       marginTop: 116,
       marginBottom: 141,
+      [theme.breakpoints.down('sm')]: {
+        marginTop: theme.spacing(6.5),
+      },
```

Comment:
Can we flip the breakpoints around to only use upwards breakpoints to maintain our mobile-first approach?